playground/frontend/playground_components/assets/translations/en.yaml

```diff
@@ -85,8 +85,7 @@ widgets:
   pipelineOptions:
     options: 'Options'
     raw: 'Raw'
-    error: >
-      Please check the format (example: --key1 value1 --key2 value2), only alphanumeric and \",*,/,-,:,;,',. symbols are allowed
+    error: 'Please check the format (example: --key1 value1 --key2 value2)'
```

Comment:
Is the alphanumeric restriction no longer true?

Reply:
Yes, it is. We decided that any characters can be used